monkey/infection_monkey/exploit/tools/payload_parsing.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import textwrap
+
+LOG = logging.getLogger(__name__)
+
+
+class Payload(object):
+    """
+    Class for defining and parsing a payload (commands with prefixes/suffixes)
+    """
+
+    def __init__(self, command, prefix="", suffix=""):
+        """
+        :param command: command
+        :param prefix: commands prefix
+        :param suffix: commands suffix
+        """
+        self.command = command
+        self.prefix = prefix
+        self.suffix = suffix
+
+    def get_full_payload(self, command=""):
```

Comment:
Perhaps rename to `get_formatted_payload`? Because when using it in LimitedSizePayload, you're actually NOT getting the full payload here...

Reply:
I don't know how to make it more readable. Maybe split into separate functions: 1 for adding payload's suffix and prefix for any command and 1 for adding payload's suffix and prefix to payload object's command. What do you think?